lunary/__init__.py

```diff
@@ -737,15 +740,15 @@ def _parse_output(raw_output: dict) -> Any:
         answer_value = raw_output.get("answer")
         result_value = raw_output.get("result")
 
-        if text_value:
+        if text_value is None:
```

Comment:
i just realize that these logic are wrong. They should be `is not None`. I submitted #37  to fix this 🙇 .